src/foam/nanos/approval/refinements.js

```diff
@@ -1,6 +1,6 @@
 foam.CLASS({
     package: 'foam.nanos.approval',
-    name: 'UserRefines',
+    name: 'refinements',
```

Comment:
```suggestion
    name: 'UserRefinement',
```
The class name should be capitalized.